### PR TITLE
Add timezone

### PR DESCRIPTION
**Pull request by tank157**
Part of the work for [this Pivotal Card](https://www.pivotaltracker.com/story/show/89524224)

In order to communicate realistic times to users, we needed to get the timezone so we can display it in such.

### CODE REVIEW

File: src/usps_processor/parse.clj [hunk 1]
```diff
@@ -1,15 +1,53 @@
...
+            [usps-processor.zip-lookup :as zip]))
+
+(def default-timezone-id "America/Los_Angeles")
+(def usps-time-format-pattern "M/d/yyyy H:mm:ss")
+
+(defn zip->timezone-id
+  "Looks up the id of the timezone for the provided zip."
+  [zip]
+  (or (some-> zip
+              (Integer/parseInt)
```
Comment (tie-rack) [inline src/usps_processor/parse.clj]: In a very real way, zip codes are not integers, they are strings of numbers. Treating them as numbers seems like a way to get into trouble.

File: src/usps_processor/zip_lookup.clj [hunk 2]
```diff
@@ -13,11 +13,12 @@
...
+  (let [data (-> "zip_city_state_timezone.edn"
                  clojure.java.io/resource
                  slurp
                  edn/read-string)]
     (into {}
-          (for [[zip {:keys [city state]}] data]
-            [zip {:city (proper-case city) :state state}]))))
+          (for [[zip {:keys [city state timezone-id]}] data]
+            [zip {:city (proper-case city) :state state
+                  :timezone-id timezone-id}]))))
```
Comment (tie-rack) [inline src/usps_processor/zip_lookup.clj]: Is there a reason to not have the data in "zip_city_state_timezone.edn" be already in the state we want to use it?

**Comment by tie-rack** [comment]
There appear to be a lot of entries in the zip_city_state_timezone.edn file without time zones that we should know (e.g., the very first one). Should we worry about having more complete data?

File: src/usps_processor/parse.clj (hunk 1 above)
Comment (ddellacosta) [inline src/usps_processor/parse.clj]: Fixed!

**Comment by tank157** [comment]
We started with a third party database that provided zip code and timezone ids (among a lot of other data) and filled in where we had a match and it had a time zone. I don't really know what we should be doing about filling in data where the data sets don't line up, or where there are holes even when we do match up. We'll probably need to address the gaps we have at some point.

File: src/usps_processor/zip_lookup.clj (hunk 2 above)
Comment (tank157) [inline src/usps_processor/zip_lookup.clj]: Possibly, but that wasn't a substantive change with this PR, assuming you're referring to the use of `proper-case`, and we're far removed in time (and possibly personnel) from the time that decision was made. I'm all for cleaning up technical debt when we can, but this seems like a good time to put in a card to deal with it later when we have free time.

**Comment by tie-rack** [comment]
Okay. LGTM with those future stories.

**Pull request merged**